isis/tests/data/isisdata/mockup/hayabusa/kernels/spk/hay_osbj_050911_051118_v1n.bsp

```diff
@@ -0,0 +1,9 @@
+{
+    "source": "$ISISDATA/hayabusa/kernels/spk/hay_osbj_050911_051118_v1n.bsp",
+    "filesize": 81458176,
+    "createtime": "2022-08-12T22:29:59.000000",
+    "modifiedtime": "2022-11-23T23:37:37.651359",
+    "md5hash": "bbc5587a59fd5d5d9683ac5d957649ce",
+    "sha256hash": "8577f45d89d376a9bb1351b89e53c8d4e26327e465cc70b3bf361c975db53515",
+    "sha1hash": "072e199a7b2697b97e79c1fa6aa1a186b5e03dc3"
+}
```

Comment:
> Do the tests depend on having real binary kernels?

Only to compute (meaningful) hash values, which is why there is a reference to the real kernel/file.

> Could the test setup not simply create an empty kernel?

It could. But that does not test the validation aspect of this application. You could (and highly recommended to) pick kernels which you know its file name, size and contents should not change or file not exist (all part of verification) after any download/install. Modify the test to only compute hashes for these kernels/files and it automatically validates your environment. I don't know how to make a better/useful test.

> Part of the application is comparing hashes, is this hash comparison critical to the apps tests?

Yes. This validates $ISISDATA. By definition, this framework should also be completely consistent with conditions in any valid (i.e., fully downloaded/populated) ISIS test environment since it requires $ISISDATA to run the tests.

There are two things I cannot/do not test: file times and symbolic links since support/results have proven to be inconsistent across OSes for these file/directory/device characteristics. However, these data have proven to be useful on local systems, particularly when creating/configuring new mission kernel support systems in ISIS.